packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -27,9 +32,9 @@
     ref="cardContainer"
     :class="['card-container', addClass]"
   >
-    <div :class="['card', { 'expandable-card': isExpandableCard }, borderType]">
+    <div :class="['card', { 'expandable-card': isExpandableCard, 'card-seamless': isSeamless }, borderType]">
       <div
-        :class="['card-header',{'header-toggle':isExpandableCard}, cardType, borderType]"
+        :class="['card-header', {'header-toggle':isExpandableCard}, cardType, borderType]"
```

Comment:
Unnecessary change

Reply:
Sorry about that, have made the changes accordingly